libraries/r36-buildtime/Dockerfile

```diff
@@ -0,0 +1 @@
+COPY --chown=0:0 r36-buildtime/context/build /usr/local/bin/algorithmia-build
```

Comment:
You want to be copying over the other parts of what are currently in the `bin` dir into here so it's not in the source repos